resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/kubernetes/submit/v2/ClientV2Suite.scala

```diff
@@ -88,8 +100,30 @@ class ClientV2Suite extends SparkFunSuite with BeforeAndAfter {
     submissionKubernetesClient = mock[KubernetesClient]
     podOperations = mock[PODS]
     resourceListOperations = mock[RESOURCES]
-    mountedDependencyManagerProvider = mock[MountedDependencyManagerProvider]
-    mountedDependencyManager = mock[MountedDependencyManager]
+    submittedDependencyManagerProvider = mock[SubmittedDependencyManagerProvider]
+    mountedDependencyManager = mock[SubmittedDependencyManager]
+    remoteDependencyManagerProvider = mock[DownloadRemoteDependencyManagerProvider]
+    remoteDependencyManager = mock[DownloadRemoteDependencyManager]
+    when(remoteDependencyManagerProvider.getDownloadRemoteDependencyManager(any(), any(), any()))
+      .thenAnswer(new Answer[DownloadRemoteDependencyManager] {
```

Comment:
can you use `ArgumentCaptor.forClass` instead of `any()` to eliminate this custom `Answer` implementation?